dask/array/overlap.py

```diff
@@ -203,7 +219,7 @@ def trim_internal(x, axes, boundary=None):
                       x, chunks=chunks, dtype=x.dtype)
 
 
-def _trim(x, axes, boundary, block_info):
+def _trim(x, axes, boundary, block_info):  #TODO Add support for assymetrical axes
```

Comment:
What is not handled here?

Reply:
I think that there will need to be additional work to support cases where (-/+) tuples are passed as overlap depth dimensions. In cases where boundary is not None.

Reply:
Since this is basically the same problem as the one above. We can keep the discussion up there for simplicity.